requirements.txt

```diff
@@ -1,6 +1,8 @@
-matplotlib==3.3.3
+pytest==6.2.3
```

Comment:
This requirements.txt is used to install all dependencies needed for Francis, but another requirements.txt is available in the tests folder to install dependencies for dev. 
If you need to fix a version for pytest, feel free to change it in the other requirement.txt file

Reply:
ok nice !